near-sdk-sim/src/outcome.rs

```diff
@@ -100,10 +96,10 @@ impl ExecutionResult {
     }
 
     fn get_outcome(&self, hash: &CryptoHash) -> Option<ExecutionResult> {
-        match (*self.runtime).borrow().outcome(hash) {
-            Some(out) => Some(ExecutionResult::new(out, &self.runtime, hash.clone())),
-            None => None,
-        }
+        (*self.runtime)
+            .borrow()
+            .outcome(hash)
```

Comment:
hmm, I guess clippy didn't complain about this unnecessary explicit `deref`

Reply:
it is needed here though, to indicate that it's borrowing through `RefCell` and not the `Rc`. There are other ways to indicate this, but I don't want to make any changes that can be potentially breaking if possible in this PR.